circuit_knitting/forging/entanglement_forging_knitter.py

```diff
@@ -191,6 +207,8 @@ def __call__(
         tensor_ansatze_u = [
             prep_circ.compose(circuit_u) for prep_circ in self._tensor_circuits_u
         ]
+        if self.fixed_hf_value is not None:
+            tensor_ansatze_u = tensor_ansatze_u[1:]
```

Comment:
I don't fully understand this (and the similar lines with `tensor_ansatze_v`, below).

Reply:
We know we are going to ignore the subexperiments relating to the hartree fock bitstring, so we just cut off the circuit which represents that bitstring and ignore it here. Same below, if the two subsystems are to be evaluated independently

Reply:
OK, that makes sense.  Thanks.